src/app/components/selected-items/SelectedItem.jsx

```diff
@@ -16,7 +16,7 @@ function SelectedItem({ id, name, removeClassNames, classNames, handleRemoveItem
 SelectedItem.propTypes = {
     id: PropTypes.string.isRequired,
     name: PropTypes.string.isRequired,
-    onRemoveItem: PropTypes.func.isRequired,
```

Comment:
Do the following places also need an update of of the prop `onRemoveItem` -> `handleRemoveItem` https://github.com/ONSdigital/florence/blob/9880d4843691fec453d4ad616244976e27c6c33a/src/app/views/collections/edit/CollectionEdit.jsx#L144

https://github.com/ONSdigital/florence/blob/11f5fb6fa1d61932f6ec9728bae40abfa6aedd3b/src/app/views/users/groups/User.jsx#L16

https://github.com/ONSdigital/florence/blob/c3a0476d1e27c85cbccfc265894d980bd6b50961/src/app/views/collections/create/CreateNewCollection.jsx#L320